api/http/router.go

```diff
@@ -30,12 +30,12 @@ import (
 
 // API URL used by the HTTP router
 const (
-	APIURIDevices    = "/api/devices/v1/deviceconfig"
-	APIURIInternal   = "/api/internal/v1/deviceconfig"
-	APIURIManagement = "/api/management/v1/deviceconfig"
+	URIDevices    = "/api/devices/v1/deviceconfig"
```

Comment:
where is that in the commit message? (nitpick mode continued)

Reply:
I have a bad habit of sometimes renaming things and forgetting about it. (Sorry)